server/backend/courses/tests/test_serializers.py

```
@@ -0,0 +1,102 @@
+
+    def test_course_serializer_invalid_data(self):
+        # Missing required fields like title should raise ValidationError
+        invalid_data = {}
+        serializer = CourseSerializer(data=invalid_data)
+        assert not serializer.is_valid()
+        assert "title" in serializer.errors
+
+@pytest.mark.django_db
+class TestCourseContentsSerializer:
+
+    def test_course_contents_serializer_valid(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_course_contents_serializer_valid`

Add return type annotation: `None`

(ANN201)